fili-core/src/main/java/com/yahoo/bard/webservice/web/ResponseData.java

```diff
@@ -88,7 +87,8 @@ public Response(
             SimplifiedIntervalList volatileIntervals,
             Map<String, URI> paginationLinks,
             Pagination pagination,
-            ObjectMappersSuite objectMappers
+            ObjectMappersSuite objectMappers,
+            ApiRequest apiRequest
```

Comment:
I really hesitate to add `apiRequest` to ResponseData, but it's need by write function in [line 183](https://github.com/tarrantzhang/fili/blob/bca02a125026e5568c077c6f46fac4620ecbaf00/fili-core/src/main/java/com/yahoo/bard/webservice/web/ResponseData.java#L183)